### PR TITLE
Show all organizations on auth profile page. Closes #96

### LINKED ISSUE #96: I have some organizations, but not appeared
There are no any organizations on `ORGANIZATIONS` section on my profile page. But I have some orgs. 

It seems a bug? right?

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

File: src/auth/auth.action.js [hunk 1]
```diff
@@ -51,14 +53,24 @@ export const getUser = () => {
...
+        const orgs = [];
+
+        data.forEach(part => part.map(org => orgs.push(org)));
+
         dispatch({
           type: GET_AUTH_ORGS.SUCCESS,
-          payload: data,
+          payload: uniqby(orgs, 'id').sort(
+            (org1, org2) => org1.login > org2.login
+          ),
```
Comment (andrewda) [inline src/auth/auth.action.js]: GitHub sorts orgs on profiles by the creation date of the organization. I like alphabetically more, but I'm not sure if anyone feels strongly about trying to mimic GitHub?

File: src/auth/auth.action.js [hunk 2]
```diff
@@ -51,14 +53,24 @@ export const getUser = () => {
...
 
-    fetchAuthUserOrgs(accessToken)
+    Promise.all([
+      fetchAuthUserOrgs(accessToken),
+      fetchUserOrgs(login, accessToken),
+    ])
       .then(data => {
+        const orgs = [];
+
+        data.forEach(part => part.map(org => orgs.push(org)));
```
Comment (housseindjirdeh) [inline src/auth/auth.action.js]: Probably simpler to just do `const orgs = data[0].concat(data[1])`

File: src/auth/auth.action.js (hunk 1 above)
Comment (housseindjirdeh) [inline src/auth/auth.action.js]: I don't mind alphabetically either actually 👍

File: src/auth/auth.action.js (hunk 2 above)
Comment (lex111) [inline src/auth/auth.action.js]: Exactly, I completely forgot :confused:

**Comment by lex111** [comment]
All fixed, also added a note:
image

P.S. And can it be better to take `yarn.lock` in .gitignore file?

**Comment by andrewda** [comment]
@lex111 I think the `yarn.lock` isn't supposed to be ignored, but could be wrong about that.

**Comment by housseindjirdeh** [comment]
Yep you definitely want to [check in yarn.lock files](https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/)

Thanks @lex111 this looks amazing 🎉

**Comment by lex111** [comment]
@andrewda @housseindjirdeh ok, thanks for the links with the explanations.

**Pull request merged**